resources/wiki-overrides.properties

```diff
@@ -17,19 +14,16 @@ chef-tracking=https://wiki.jenkins-ci.org/display/JENKINS/Chef+Tracking+Plugin
 chosen-views-tabbar=https://wiki.jenkins-ci.org/display/JENKINS/Chosen+Views+Tab+Bar
 ## Source seems to *not* be under jenkinsci org: https://github.com/ThoughtsOnMobile/cocoapods-jenkins-integration
 cocoapods-integration=https://wiki.jenkins-ci.org/display/JENKINS/CocoaPods+Plugin
-cucumber-reports=https://wiki.jenkins-ci.org/display/JENKINS/Cucumber+Reports+Plugin
 custom-view-tabs=https://wiki.jenkins-ci.org/display/JENKINS/Custom+View+Tabs+Plugin
 ## Currently has the <url> tag, BUT with a typo https://wiki.jenkins-ci.org/display/JENKINS/Deploment+Notification+Plugin
 deployment-notification=https://wiki.jenkins-ci.org/display/JENKINS/Deployment+Notification+Plugin
 database-drizzle=https://wiki.jenkins-ci.org/display/JENKINS/Drizzle(+MySQL)%20Database%20Plugin
 diskcheck=https://wiki.jenkins-ci.org/display/JENKINS/DiskCheck+Plugin
-docker-build-publish=https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Docker+Build+and+Publish+plugin
```

Comment:
https://github.com/jenkinsci/docker-build-publish-plugin/commit/0240b543c668767a85b20ba9b9ef839c696a2244 references the same URL.